hw2/iblis17/q2/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.py[oc]
```

Comment:
don't upload garbage.

Reply:
well, I think the committing the gitignore file is acceptable in a git repo.

Reply:
> excluding any irrelevant files, such as binaries, text files, or dot files

Reply:
yes. gitignore is important.
However, you should only upload code for assigments.

Reply:
:ok_hand: